package.json

```diff
@@ -3,16 +3,16 @@
   "version": "0.0.1",
   "description": "Project Platform Health",
   "scripts": {
-    "build": "neutrino build",
-    "start": "neutrino start",
-    "start:local": "BACKEND=http://localhost:3000 neutrino start",
-    "lint": "neutrino lint",
+    "build": "BACKEND=https://firefox-health-backend.herokuapp.com webpack --optimize-minimize",
+    "start": "BACKEND=https://firefox-health-backend.herokuapp.com webpack-dev-server --mode development",
+    "start:local": "BACKEND=http://localhost:3000 webpack-dev-server --mode development",
```

Comment:
If you wanted to avoid having to put `BACKEND` here, a default can be set via the preset's `env` option:
https://master.neutrinojs.org/packages/web/#environment-variables